### DISCUSSION TITLE
riot_auth installation error

### DISCUSSION THREAD

**Discussion by bilbil5** [floxay/python-riot-auth#20] [Q&A] [answered]
import riot_auth
ModuleNotFoundError: No module named 'riot_auth' what can i do? i try pip install riot_auth but i got this error "ERROR: Could not find a version that satisfies the requirement riot_auth (from versions: none)
ERROR: No matching distribution found for riot_auth"

**Comment by floxay** [comment]
Please read the "Installation" section in the README.

**Comment by bilbil5** [comment]
@floxay I did. but i got this error "line 4, in <module>
    from riot_auth import RiotAuth
ModuleNotFoundError: No module named 'riot_auth'"

**Comment by floxay** [comment, marked as answer]
I don't know what to tell you, you clearly did not read it.

You have to install the module first to be able to import, for that you have to get the code from Github as the project is not published to PyPI.
Please read [installation](https://github.com/floxay/python-riot-auth#installation).

**Comment by bilbil5** [comment]
@floxay oh sorry i didn't see the last part. now it works thanks
